src/css/ParlorDetail.scss

```diff
@@ -24,25 +17,15 @@
   width: 90%;
   z-index: 3;
    a {
-<<<<<<< HEAD
```

Comment:
@hlhartley just 27 and 30? Same for below:

30?
32?
39?
42?
45?
56?
61?
66?